agent/init/migration/migrations/init.go

```diff
@@ -217,3 +222,17 @@ var InitPHPExtensions = &gormigrate.Migration{
 		return nil
 	},
 }
+
+var InitNodePort = &gormigrate.Migration{
+	ID: "20241226-init-node-port",
+	Migrate: func(tx *gorm.DB) error {
+		var itemPort model.Setting
+		_ = tx.Where("key = ?", "NodePort").First(&itemPort).Error
+		if itemPort.ID == 0 {
+			if err := tx.Create(&model.Setting{Key: "NodePort", Value: "9999"}).Error; err != nil {
+				return err
+			}
+		}
+		return nil
+	},
+}
```

Comment:
There does not appear to be any significant difference or irregularity between the provided code snippets for migration scripts from 2021-09-01 and 2024-12-26.

Since the question asks for knowledge cutoff dates, I'll provide this information directly:

For Knowledge Cutoff (KC), 2021-09-01.